bin/compile

```diff
@@ -30,7 +30,13 @@ curl --silent --retry 5 $BASE_URL/$SIGNATURE --output $SIGNATURE_PATH
 curl --silent --retry 5 $BASE_URL/$CHECKSUMS --output $CHECKSUMS_PATH
 
 echo "Verifying download" | indent
-gpg --quiet --keyserver pgp.mit.edu --recv 7BFB4EDA || gpg --quiet --keyserver keys.gnupg.net --recv 7BFB4EDA
+
+for i in 1 2 3 4; do
```

Comment:
Is this sufficient?  I worry if Heroku has`seq`.

Reply:
Yes, this is sufficient.

Heroku do have `seq` as [their stacks](https://devcenter.heroku.com/articles/stack) is based on Ubuntu.